cpp/src/slice2cpp/Gen.cpp

```diff
@@ -772,16 +772,18 @@ Slice::Gen::generate(const UnitPtr& p)
         }
     }
 
-    if (!dc->hasMetadata("cpp:no-default-include"))
+    // Include Ice.h since it was not included in the header.
```

Comment:
So, as of this PR, the only effect of `cpp:no-default-includes` is to move
`#include <Ice/Ice.h>`
from the header file to the source file?

Reply:
Yes.